src/en/firstkissmanganet/build.gradle

```
@@ -2,7 +2,7 @@ ext {
     extName = '1st-Kiss Manga.net'
     extClass = '.FirstKissMangaNet'
     themePkg = 'madara'
-    baseUrl = 'https://1st-kissmanga.net'
+    baseUrl = 'https://1stkissmanga.org/'
```

Remove trailing slash
```suggestion
    baseUrl = 'https://1stkissmanga.org'
```